WinUIGallery/WinUIGallery.csproj

```diff
@@ -32,7 +32,9 @@
         <SingleProject Condition="'$(SolutionName)' == 'WinUIGallery.DesktopWap'">false</SingleProject>
         <WindowsAppSdkIncludeVersionInfo>true</WindowsAppSdkIncludeVersionInfo>
         <PublishProfile>win-$(Platform).pubxml</PublishProfile>
-        <PublishAot>false</PublishAot>
+		<DisableRuntimeMarshalling>true</DisableRuntimeMarshalling>
```

Comment:
Why do we need this?

Reply:
disables the built-in .NET runtime marshalling infrastructure, which is enabled by default to support [DllImport] and other interop scenarios at runtime.
I've seen this in various projects including the following:
https://github.com/microsoft/XamlBehaviors/blob/a98c24a6a7d9bdd57cf5d406455b6f623e321a72/src/BehaviorsSDKManaged/Microsoft.Xaml.Interactivity.WinUI/Microsoft.Xaml.Interactivity.WinUI.csproj#L12